React/tic-tac/src/app/screens/game/components/board/index.js

```diff
@@ -1,8 +1,9 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
-import Square from '../square/index.js';
-import style from '../board/style.scss';
+import Square from '../Square/index.js';
```

Comment:
Move the Square component to a components folder within the Board component folder. This is done to ensure that you can move the component folder to another project and it can work as expected.

Reply:
Also, keep in mind that components folders should be capitalized so it would be easier to identify React components within your app structure.